src/runtime/virtcontainers/sandbox.go

```diff
@@ -2109,7 +2113,7 @@ func (s *Sandbox) cgroupsDelete() error {
 		return nil
 	}
 
-	sandboxCgroup, err := cgroups.Load(s.state.SandboxCgroupPath)
+	sandboxCgroup, err := cgroups.LoadCgroup(s.state.SandboxCgroupPath)
```

Comment:
That call is Linux specific, though?

Reply:
This is replaced with

```golang
sandboxController, err := resCtrl.LoadResourceController(s.state.SandboxCgroupPath)
```

which is arguably less Linux specific, hopefully.